pandas/tests/scalar/test_nat.py

```diff
@@ -302,14 +302,14 @@ def test_nat_arithmetic_index():
         tm.assert_index_equal(left - right, exp)
         tm.assert_index_equal(right - left, exp)
 
-    # timedelta
+    # timedelta # GH#19124
```

Comment:
can you add similar testing for Series here as well

Reply:
Shoot this surfaces a new bug; I'll xfail for now.